### PR TITLE
Integrate SLURM in PySlowFast

### PR DESCRIPTION
**Pull request by sdebnathusc**

### CODE REVIEW

File: slowfast/config/defaults.py [hunk 1]
```diff
@@ -730,6 +731,9 @@
 # Number of GPUs to use (applies to both training and testing).
 _C.NUM_GPUS = 1
 
+# Maximum number of GPUs available per node (unlikely to need to be changed)
+_C.MAX_GPUS_PER_NODE = 8
```
Comment (lyttonhao) [inline slowfast/config/defaults.py:735]: we could remove this config?

Comment (vaibhava0) [inline slowfast/config/defaults.py:735]: This might not be required.

**Review by vaibhava0 — COMMENTED** [review]
Looks great. Thanks!

**Review by lyttonhao — CHANGES_REQUESTED** [review]
Fix num_shareds

**Review by vaibhava0 — APPROVED** [review]

**Pull request merged**